
We just place a stub wchar.h in the dietlibc include directory to get stuff
like gnu tar to compile.

  - Rene Rebe <rene@rocklinux.org>

--- dietlibc-0.27/include/wchar.h.orig	1970-01-01 01:00:00.000000000 +0100
+++ dietlibc-0.27/include/wchar.h	2004-10-02 17:45:44.051302108 +0200
@@ -0,0 +1,6 @@
+#ifndef _WCHAR_H_
+#define _WCHAR_H_
+
+/* just a stub -ReneR */
+
+#endif /* _WCHAR_H_ */
